Bandwidth.Standard/Voice/Bxml/Bxml.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace Bandwidth.Standard.Voice.Bxml
+{
+  /// <summary>
+  ///   Bxml class for Bandwidth XML
+  /// </summary>
+  [XmlRoot("Bxml", Namespace = "")]
+  public class BXML : IXmlSerializable
+  {
+    private static readonly XmlSerializer Serializer = new XmlSerializer(typeof (BXML), "");
+    private readonly List<IVerb> _list = new List<IVerb>();
+
+        private static readonly Regex XML_REGEX = new Regex("&lt;([a-zA-Z//].*?)&gt;");
+
+        private static readonly Regex SPEAK_SENTENCE_REGEX = new Regex("<SpeakSentence.*?>.*?<\\/SpeakSentence>");
```

Comment:
CRAZY_CAPS on these. Private fields should be camel casing and prefixed with `_` as you did with `_list`.